libraries/AP_DDS/README.md

```diff
@@ -104,25 +104,28 @@ sudo apt-get install socat
 Set up your [SITL](https://ardupilot.org/dev/docs/setting-up-sitl-on-linux.html).
 Run the simulator with the following command. If using UDP, the only parameter you need to set it `DDS_ENABLE`.
 
-| Name | Description |
-| - | - |
-| DDS_ENABLE | Set to 1 to enable DDS |
-| SERIAL1_BAUD | The serial baud rate for DDS |
-| SERIAL1_PROTOCOL | Set this to 45 to use DDS on the serial port |
+| Name | Description | Default |
+| - | - | - |
+| DDS_ENABLE | Set to 1 to enable DDS, or 0 to disable | 1 |
+| SERIAL1_BAUD | The serial baud rate for DDS | 57 |
```

Comment:
```suggestion
| SERIAL1_BAUD | The serial baud rate for DDS | 115 |
```

Baud rate is listed as 115200 below, and this is what the micro-ROS agent defaults to as well.

Reply:
```
sim_vehicle.py -v copter -w --enable-dds
...
param set SERIAL1_BAUD
STABILIZE> SERIAL1_BAUD     57
```